actions/helm-lint/README.md

```diff
@@ -1,32 +1,34 @@
 # helm-lint
-This action will lint helm charts inside the `charts` folder of your repository.
+
+This action will lint Helm charts inside the `charts` folder of your repository.
 
 ## Prerequisites
 
 You need to create the lint configuration file `.github/lint-config.yaml` and configure it to your liking.
 A minimal configuration could look like this:
+
 ```yaml
 # check https://github.com/helm/chart-testing/blob/main/pkg/config/test_config.yaml for possible configurations
 target-branch: "main"
 ```
 
 ## Input Parameters
-| Name              | Required  |             Default Value             |  Type   | Description                                                                                                                              |
-|-------------------|:---------:|:-------------------------------------:|:-------:|------------------------------------------------------------------------------------------------------------------------------------------|
-| ref               |    ❌     | The default branch of your repository | string  | The ref name to checkout the repository                                                                                                  |
-| lint-config-path  |    ❌     |      ".github/lint-config.yaml"       | string  | The path to the lint configuration file (For an example see https://github.com/helm/chart-testing/blob/main/pkg/config/test_config.yaml) |
-| helm-version      |    ❌     |               "v3.4.0"                | string  | The helm version                                                                                                                         |
+
+| Name             | Required |             Default Value             |  Type  | Description                                                                                                                              |
+| ---------------- | :------: | :-----------------------------------: | :----: | ---------------------------------------------------------------------------------------------------------------------------------------- |
+| ref              |    ❌    | The default branch of your repository | string | The ref name to checkout the repository                                                                                                  |
+| lint-config-path |    ❌    |      ".github/lint-config.yaml"       | string | The path to the lint configuration file (For an example see https://github.com/helm/chart-testing/blob/main/pkg/config/test_config.yaml) |
+| helm-version     |    ❌    |               "v3.10.1"               | string | The Helm version                                                                                                                         |
 
 ## Usage
 
 ```yaml
-...
+---
```

Comment:
Please remove. `---` is a separator for different yaml resources and would not work here